cfme/tests/cloud_infra_common/test_snapshots_rest.py

```diff
@@ -108,6 +108,23 @@ def test_create_snapshot(self, vm_snapshot):
         vm, snapshot = vm_snapshot
         vm.snapshots.get(description=snapshot.description)
 
+    @pytest.mark.manual
+    def test_query_snapshots(self):
+        """Queries VM/instance snapshot using REST API.
+
+        Metadata:
+            test_flag: rest
+
+        Polarion:
+            assignee: pvala
+            casecomponent: Rest
+            caseimportance: medium
+            initialEstimate: 1/4h
```

Comment:
Please either automate now, or include testSteps and expectedResults for what REST queries should be made.

Reply:
Removed that test case, since it was already covered by other tests in the file, added a blocker instead.